app/ui/src/app/integration/edit-page/current-flow.service.ts

```diff
@@ -335,6 +335,11 @@ export class CurrentFlow {
         const action = event['action'];
         const properties = this.stringifyValues(event['properties']);
         const step = this.steps[position] || createStep();
+        for (const key of Object.keys(properties)) {
+          if (properties[key] === 'null') {
```

Comment:
@gashcrumb this seems kinda smelly.  What if a user actually want to use the value 'null' for a property?

Reply:
Yeah, it's a workaround really.  For some reason the forms now produce `"null"` instead of an empty string.  I've created https://github.com/syndesisio/syndesis/issues/1250 to track this down, not sure when this crept in.